scalding-core/src/main/scala/com/twitter/scalding/reducer_estimation/Common.scala

```diff
@@ -45,6 +82,23 @@ class ReducerEstimator {
 
 }
 
+trait HistoryReducerEstimator extends ReducerEstimator with HistoryService {
```

Comment:
it could still be a trait with a `def history: HistoryService` then in subclasses you `override def history = ...`